eman2/protocols/protocol_tomo_initialmodel.py

```diff
@@ -0,0 +1,225 @@
+# **************************************************************************
+# *
+# * Authors:     Adrian Quintana (adrian@eyeseetea.com) [1]
+# *              Arnau Sanchez  (arnau@eyeseetea.com) [1]
+# *
+# * [1] EyeSeeTea Ltd, London, UK
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+import json
+import re
+from glob import glob
+from itertools import count
+
+import numpy
+
+import pyworkflow.em as pwem
+from pyworkflow.protocol import params
+from pyworkflow.utils.path import makePath
+
+import eman2
+from eman2.convert import writeSetOfParticles, getLastParticlesParams, updateSetOfSubTomograms
+
+from tomo.protocols import ProtTomoBase
+from tomo.objects import SubTomogram, SetOfSubTomograms
+
+
+class EmanProtTomoInitialModel(pwem.EMProtocol, ProtTomoBase):
+    """
+    This protocol wraps *e2spt_sgd.py* EMAN2 program.
+
+    It will take a set of class-averages/projections and build a set
```

Comment:
I guess the protocol accepts a setOfSubTomograms in general, not especifically class-averages, but you probably know it better. However, I think they are not projections in any case, but maybe this should be confirmed

Reply:
Yep, it was the wrong help. I changed the initial model and subtomogram refinement texts for some more elaborated helps here https://github.com/scipion-em/scipion-em-eman2/pull/36/commits/df1ed7237ecdf2e47a930f08d0b5fe8ee516a487